webpack.js

```diff
@@ -41,9 +42,10 @@ webpackConfig.module.rules = Object.values(webpackRules)
 webpackConfig.plugins.push(
 	// patch webdav/dist/request.js
 	new webpack.NormalModuleReplacementPlugin(
-		/request.js/,
+		/request(\.js)?/,
```

Comment:
oof, I didn't know about this. :'D

Reply:
We could make it upstream, but https://github.com/perry-mitchell/webdav-client/issues/179